### PR TITLE
Fix hostmetrics, add ephemeral config

### PR DESCRIPTION
**Pull request by amdprophet**
* Removes hostmetrics as a separate component / package
* Adds hostmetrics on Linux
* Adds ephemeral config

### CODE REVIEW

File: assets/conf.d/ephemeral.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+extensions:
```
Comment (echlebek) [inline assets/conf.d/ephemeral.yaml:1]: Isn't this the wrong file path? It's in conf.d instead of conf.d-available

**Comment by echlebek** [comment]
Where's the hostmetrics.yaml file in the assets/conf.d-available directory? Are we generating it somehow?

File: assets/conf.d/ephemeral.yaml (hunk 1 above)
Comment (amdprophet) [reply assets/conf.d/ephemeral.yaml:1]: This is just the source path in the repository. I can change it though so it's less confusing.

**Comment by amdprophet** [comment]
> Where's the hostmetrics.yaml file in the assets/conf.d-available directory? Are we generating it somehow?

The source name is `[linux|darwin|windows].yaml` and it gets renamed:
https://github.com/SumoLogic/sumologic-otel-collector-packaging/blob/main/components/otelcol-sumo.cmake#L302-L311

**Review by echlebek — APPROVED** [review]

**Pull request merged**